src/components/StudyList/StudyList.tsx

```diff
@@ -0,0 +1,58 @@
+import { forwardRef } from 'react';
+import { StudyListType } from '@interfaces/studyList';
+import { StudyCard, SpinnerIcon } from '@components';
+
+import { Ul, ItemCard } from './StudyList.style';
+
+interface Props {
+  studyList: StudyListType;
+  error: boolean;
+  loading: boolean;
+  onStudyDelete: (studyId: number) => void;
+}
+
+export type Ref = HTMLLIElement;
+
+const StudyList = forwardRef<Ref, Props>(function StudyList(
+  { studyList, loading, error, onStudyDelete },
+  ref,
+) {
+  return (
+    <Ul>
+      {studyList.length !== 0 &&
```

Comment:
```suggestion
      {studyList?.length&&
```
이렇게도 바꿀 수 있을 것 같습니다!